hack/cluster-sync.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# ******************************************************
+# Create service account and kubeconfig to access tenant cluster
+# ******************************************************
+./kubevirtci kubectl -n kvcluster create -f ./deploy/infra-cluster-service-account.yaml
```

Comment:
this is kind of a broad suggestion for all this automation.

maybe use `apply` instead of `create`. That way if the script fails half way through for some reason, someone can run it again and only the changes will be applied.  Without using `apply` it's impossible to run this script again because the `create` will fail if the service-account already exists.

Reply:
that is very good point, I'll try to change it now

Reply:
done, it really works great